src/Lachain.Core/Blockchain/SystemContracts/Interface/Lrc20Interface.cs

```diff
@@ -13,6 +13,9 @@ public class Lrc20Interface : IContractInterface
         public const string MethodName = "name()";
         public const string MethodDecimals = "decimals()";
         public const string MethodSymbol = "symbol()";
+        public const string MethodMint = "mint(address,uint256)";
+        public const string MethodSetAllowedSupply = "setAllowedSupply(uint256)";
+        public const string MethodGetAllowedSupply = "getAllowedSupply()";
```

Comment:
I think we should return uint256 here

Reply:
we are not returning any values here, we are setting the actual values in the `frame.ReturnValue` and returning `ExecutionStatus` which I string